### PR TITLE
feat: add transactions and nft info to the history response

### LINKED ISSUE #339: feat: update account history response format
Update account history response format to provide following spec:
* Fiat value of transaction
* Crypto value of transaction (actions like swaps have 2 values)
* Url(s) to token(s) images
* Url(s) to nft image
* NFT name / Collection name
* ENS names when available?

Current spec `curl -v "https://rpc.walletconnect.com/v1/account/0xf3ea39310011333095CFCcCc7c4Ad74034CABA63/history?projectId=b5dba79e421fd90af68d0a1006caf864" | jq .`

Part of #317

### PR DESCRIPTION
**Pull request by geekbrother**
# Description

This PR adds `transfers` object to the account history response format to add:
* Fiat value of transaction
* Crypto value of transaction
* Url(s) to token(s) images
* Url(s) to nft image
* NFT name / Collection name

The new response format is:
```
data: [
 {
    "id": "1a5a22ea3cb35fdb9080c09fe7b57ecc",
      "metadata": {
        "operationType": "mint", // Enum: approve, borrow, burn, cancel, claim, deploy, deposit, execute, mint, receive, repay, send, stake, trade, unstake, withdraw
        "hash": "0x70783a3580f2eb63f3c35fdcdc8feffbe591eea1beaaec6eb4592b19bfb73e45",
        "minedAt": "2023-10-29T14:25:29Z",
        "sentFrom": "0x66eeb19f3ef93312541009f855ff4c900adb3450",
        "sentTo": "0xa2a829f1fd922f283282d0d2367f69a13167a9b1",
        "status": "confirmed", // Enum: confirmed, failed, pending
        "nonce": 3126
      }

    // New object
    "transfers": [
       {
         direction: "in", // Enum: in, out, self
         quantity: {
              numeric: "35.1234",
         },
         value: 1350,
         price: 10,
         fungible_info: {
           name: "Bankless BED Index",
           symbol: "BED",
           icon_url: "https://token-icons.s3.amazonaws.com/0x0391d2021f89dc339f60fff84546ea23e337750f.png",
         },
         nft_info: {
           name: "#10 De·genesis",
           content: {
             preview: {
               url: "https://token-icons.s3.amazonaws.com/0x1494ca1f11d487c2bbe4543e90080aeba4ba3c2b.png",
               content_type: "image/png",
             },
             detail: {
               url: "https://token-icons.s3.amazonaws.com/0x1494ca1f11d487c2bbe4543e90080aeba4ba3c2b.png",
               content_type: "image/png",
             }
           }
         }
       }
    ]
 }
]
```

The `transfers` object contains data from the [zerion wallet transactions](https://developers.zerion.io/reference/listwallettransactions).

Resolves #339 

## How Has This Been Tested?

`account_history_check` functional test in this PR.

## Due Diligence

* [ ] Breaking change
* [x] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

**Review by arein — APPROVED** [review]
awesome!

File: src/handlers/mod.rs [hunk 1]
```diff
@@ -58,3 +59,42 @@ pub struct HistoryTransactionMetadata {
...
+    pub name: String,
+    pub symbol: String,
+    pub icon_url: String,
+}
+
+#[derive(Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct HistoryTransactionTransferInfo {
+    pub fungible_info: Option<HistoryTransactionFungibleInfo>,
+    pub nft_info: Option<HistoryTransactionNFTInfo>,
```
Comment (chris13524) [inline src/handlers/mod.rs:75]: Why are these option types? Seems like the `None` variant isn't being leveraged

**Review by chris13524 — APPROVED** [review]

File: src/handlers/mod.rs (hunk 1 above)
Comment (geekbrother) [reply src/handlers/mod.rs]: Thanks for catching this. Fixed and removed duplicated structures as well.

**Review by chris13524 — APPROVED** [review]

**Comment by stvnreaves** [comment]
> awesome!

**Pull request merged**